### ISSUE TITLE
Bug: Nested media queries report when they directly use a descriptor

### ISSUE THREAD

**Issue by ldavidpace** [eslint/css#255]
### Environment

ESLint version: v9.16.0
@eslint/css version: 0.11.0
Node version: v22.19.0
npm version: v9.6.7
Operating System: darwin 24.5.0

### Which language are you using?

stylesheet

### What did you do?

<details>
<summary>Configuration</summary>

```
{
language: 'css/css',
languageOptions: {
	tolerant: true,
},
plugins: {css: css},
rules: {
'css/no-invalid-at-rules': 'error'
}
}
```
</details>

```css
.responsive {
  width: 800px;
  @media (max-width: 800px) {
     width: 500px
  }
}

```

### What did you expect to happen?

No errors reported

### What actually happened?

Unknown descriptor 'width' found for at-rule '@media'  css/no-invalid-at-rules

### Link to Minimal Reproducible Example

https://github.com/ldavidpace/NestedMediaError

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by BastiDood** [comment]
Hello there! I'd like to take this bug on. I've done some experimenting and have a fix up in [my branch](https://github.com/BastiDood/eslint-css/tree/fix/issue-255). Just waiting for this ticket to be approved before submitting the pull request.

**Comment by nzakas** [comment]
Hi folks - just a note on the process here. We need to wait until an issue is marked as "accepted" before starting work. I'll do that now, just something to keep in mind in the future. We want someone from the team to verify bugs before people open PRs to ensure we're not wasting time on something that isn't a bug.

**Comment by Tanujkanti4441** [comment]
Hi @xbinaryx, would it be helpful if I sent over a fix, or are you still working on it?
